src/06-serial-communication/nix-tooling.md

```diff
@@ -19,42 +19,9 @@ the board; check the next section, which contains instructions for older
 revisions. If you do have a newer revision skip the next section and move to the
 "minicom" section.
 
-## Older revisions of the discovery board / external serial module
+## minicom
 
-Connect the serial module to your computer and let's find out what name the OS assigned to it.
-
-> **NOTE** On macs, the USB device will named like this: `/dev/cu.usbserial-*`. You won't
-> find it using `dmesg`, instead use `ls -l /dev | grep cu.usb` and adjust the following 
-> commands accordingly!
```

Comment:
This is still useful for macOS

Reply:
In theory it's possible that this info might be wrong with the micro:bit so I removed it (I cannot check whether it's true since I dont own a Mac) but let's put it back in until proven wrong ^^

Reply:
Actually reading through the old version again, this part of the document is named " ## Older revisions of the discovery board / external serial module" so it's probably irrelevant if the stuff before this chapter already works? Would be cool if someone with a Mac could verify this.

Reply:
I will check on it

Reply:
THis mac behaviour isnt relevant anymore according to therealprof.

Reply:
Well, I said the `/dev/cu.*` isn't relevant. It might still be good to know how to find the device node... Not sure whether this is mentioned somewhere else. ;)